develop-docs/self-hosted/index.mdx

```diff
@@ -25,26 +25,26 @@ sudo ./install.sh
 
 ## Recommended system resource
 
-Depending on your usage, required system resource to run Sentry may varies. Minimum system specification for running Sentry looks like this:
-
-- 2 CPU cores
-- 4 GB RAM
-
-Having a machine that have lower system specification than that will fail the installation script.
-
-Current recommended system specification for running Sentry is:
+Depending on your usage, required system resource to run Sentry may varies. Current recommended system specification for running Sentry is:
```

Comment:
We now have these are system minimum requirements, so should update this from recommended -> required